src/util/helpers.tsx

```diff
@@ -335,3 +336,19 @@ export const getDefaultStoragePool = (profile: LxdProfile) => {
     });
   return rootStorage ? rootStorage.pool : "";
 };
+
+export const base64EncodeTokenDetails = (tokenPreResponse: LxdTrustToken) => {
+  const tokenDetails = {
+    client_name: tokenPreResponse.client_name,
+    fingerprint: tokenPreResponse.fingerprint,
+    addresses: tokenPreResponse.addresses,
+    secret: tokenPreResponse.secret,
+    expires_at: tokenPreResponse.expires_at,
+    type: tokenPreResponse.type,
+  };
+
+  const jsonString = JSON.stringify(tokenDetails);
+  const base64Encoded = btoa(jsonString);
+
+  return base64Encoded;
+};
```

Comment:
```suggestion
export const encodeBase64Token = (data: unknown) => {
  const jsonString = JSON.stringify(data);
  return btoa(jsonString);
};
```

We shouldn't couple a general concept like base64 encoding to the tls identities data.